tensorflow_graphics/projects/pointnet/train.py

```diff
@@ -130,28 +129,27 @@ def evaluate():
 # ------------------------------------------------------------------------------
 # ------------------------------------------------------------------------------
 
-if not FLAGS.dryrun:
-  ds_train, info = ModelNet40.load(split="train", with_info=True)
-  num_examples = info.splits["train"].num_examples
-  ds_train = ds_train.shuffle(num_examples, reshuffle_each_iteration=True)
-  ds_train = ds_train.repeat(FLAGS.num_epochs)
-  ds_train = ds_train.batch(FLAGS.batch_size)
-  ds_test = ModelNet40.load(split="test").batch(FLAGS.batch_size)
+ds_train, info = ModelNet40.load(split="train", with_info=True)
+num_examples = info.splits["train"].num_examples
+ds_train = ds_train.shuffle(
+    num_examples, reshuffle_each_iteration=True)
```

Comment:
not necessary newline (#hate newlines)

Reply:
got around to configuring formatter. It makes `sys.argv` take up a lot of lines, but it's consistent.

Reply:
No worries, we'll fix this internally